skimage/segmentation/manual_segmentation.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.patches import Polygon
+from matplotlib.collections import PatchCollection
+from ..draw import polygon
+
+
+def manual(image, alpha=0.4):
+    """Return a binary image based on the selections made with mouse clicks.
+
+    Parameters
+    ----------
+    image : (M, N[, 3]) array
+        Grayscale or RGB image.
+
+    alpha : float or None
+        Transparancy value for polygons draw over the segments.
+
+    Returns
+    -------
+    mask : (M, N) array
+        Boolean image with segmented regions.
+
+    Notes
+    -----
+    Use the cursor to draw objects.
+
+    Examples
+    --------
+    >>> from skimage import data, segmentation, io
+    >>> camera = data.camera()
+    >>> mask = segmentation.manual(camera)
+    >>> # Use the cursor to draw objects
+    >>> io.imshow(mask)
```

Comment:
These doctests fail currently. Add the string `  # doctest: +SKIP` after `mask = ...` and after each following line.